### PR TITLE
Newuserexp

### PR DESCRIPTION
**Pull request by amcjen**
New user experience YO!

### CODE REVIEW

File: Apps/dashboardv3/dashboard-client.js [hunk 1]
```diff
@@ -61,6 +62,23 @@ app.configure(function() {
     });
 });
 
+function checkInstalled(req, res, next) {
+    if (connectPage === false) {
+        // TODO: fix this to redirect upon new locker w/ no services connected
+        locker.mapType("connector", function(err, installedConnectors) {
```
Comment (amcjen) [inline Apps/dashboardv3/dashboard-client.js]: Jer, this isn't the correct call it seems.  It looks like mapType just calls /map internally, but maybe "connector" is incorrect now with imthemap changes?  Digging still.

File: Connectors/LinkedIn/linkedin.synclet [hunk 2]
```diff
@@ -0,0 +1,10 @@
```
Comment (quartzjer) [inline Connectors/LinkedIn/linkedin.synclet:0]: this crept back in somehow, shouldn't be here :)

File: Connectors/Twitter/migrations/1345636562997_remove_twitter_me.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
```
Comment (quartzjer) [inline Connectors/Twitter/migrations/1345636562997_remove_twitter_me.js:0]: this also crept back in somehow?

File: Apps/dashboardv3/static/js/connect.js [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+        popup.focus();
+    });
+    
+    if ($('.sidenav-items.synclets .installed', window.parent.document).length > 0) {
+        showAllConnectors();
+    }
+    
+    $('#start-exploring-link').click(function(e) {
+        e.preventDefault();
+        window.parent.location.hash = "#contactsviewer";
```
Comment (quartzjer) [inline Apps/dashboardv3/static/js/connect.js]: this is the default app to open? is this "default" referenced in multiple places?

File: Apps/dashboardv3/static/js/connect.js [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+    }
+};
+
+var updateUserProfile = function() {
+    
+    var username = null;
+    var avatar = null;
+    
+    var fetchUserProfile = function() {
+        $.get('/synclets/facebook/get_profile', function(body) {
```
Comment (quartzjer) [inline Apps/dashboardv3/static/js/connect.js:70]: this happened after you wrote this code so I don't think you have to update it (probably file an issue instead), but there is /map/profiles now which returns an object containing all of the profiles connected to the system (hit it locally to see the format), get_profile is depreciated :)

Comment (amcjen) [inline Apps/dashboardv3/static/js/connect.js:70]: Awesome!  Much cleaner.  I'll swap over to using the new way.

**Comment by quartzjer** [comment]
this worked great for me locally, thanks Eric, big +1!

**Pull request merged**